HACKING.rst

```diff
@@ -13,6 +13,7 @@ Dependencies
 
 Ensure you have ``python-apt`` installed in your development environment.
 On Ubuntu, the correct source package for your distro can be found at https://launchpad.net/ubuntu/+source/python-apt
+To get the required version into your ``venv``, run ``tox`` once.  Alternatively, ``pip install`` the ``python-apt`` version at the URL listed in the ``tox.ini`` section ``[test]``.  After that, you can run ``pytest`` as normal.
```

Comment:
This might be a good opportunity to mention that `tox devenv` can do this all for you if you use the right environment name. https://tox.wiki/en/4.13.0/cli_interface.html#tox-devenv-(d)

Reply:
For me at least, tox will not install in the current venv:
```
tox
ROOT: will run in automatically provisioned tox
ROOT: provision> .tox/.tox/bin/python -m tox
```

But yes, `tox devenv` is the better way to create a venv if you don't want to do it manually.

Reply:
@syu-w you're right - running `tox` by default will create new virtual environments just for tox usage, so `tox devenv` or manually installing the correct version of `python-apt` are the correct ways to go.

Reply:
@lengau @syu-w Github didn't show me your comment replies before I posted mine.

I see what you guys are saying, I'll update.

Reply:
@lengau @syu-w Updated the PR!